tests/feature_elimination/test_feature_elimination.py

```diff
@@ -72,6 +73,58 @@ def test_shap_rfe(X, y, capsys):
     out, _ = capsys.readouterr()
     assert len(out) == 0
 
+def test_shap_rfe_cols_to_keep(X, y, capsys):
+    """
+    Test for shap_rfe_cv with feautures to keep parameter.
+    """
+    clf = DecisionTreeClassifier(max_depth=1,random_state=1)
+    with pytest.warns(None) as record:
+        shap_elimination = ShapRFECV(clf, random_state=1, step=1, cv=2, scoring='roc_auc', n_jobs=4,min_features_to_select=1)
+        shap_elimination = shap_elimination.fit(X, y,columns_to_keep=['col_2'])
+
+    assert shap_elimination.fitted == True
+    shap_elimination._check_if_fitted()
+
+    report = shap_elimination.compute()
+
+    assert report.shape[0] == 3
+    reduced_feature_set = set(shap_elimination.get_reduced_features_set(num_features=1))
```

Comment:
Can we check if lists match? Checking sets match could hide if there are duplicates (unlikely)